pycrostates/utils/mixin.py

```diff
@@ -2,19 +2,20 @@
 
 from mne.io.meas_info import ContainsMixin as MNEContainsMixin
 from mne.io.meas_info import MontageMixin as MNEMontageMixin
-
+from mne.channels.channels import SetChannelsMixin as MNESetChannelsMixin
+from mne.channels.channels import UpdateChannelsMixin as MNEUpdateChannelsMixin
 from ._docs import copy_doc
 
 
-class ChannelsMixin:
+    
+class ChannelsMixin(MNESetChannelsMixin, MNEUpdateChannelsMixin):
     """Channels Mixin for futur implementation."""
 
     # TODO: Maybe some part are salvageable from mne.channels.channels:
     # - UpdateChannelsMixin
     # - SetChannelsMixin
     pass
```

Comment:
Let's be cautious here, I don't remember which class inherits from ChannelsMixin in `pycrostates` if any; but we need to make sure this does not break any existing code (or add broken methods to them).

Reply:
yep I can confirm that.

I started to test and implement all the methods from thoses classes with `ch_data` but there is a **huge amount** of work here.
I remove the support of `ChData` instances in `apply_spatial_filter` for now.  One can apply the spatial filter before converting it to `ChData` anyway.

Reply:
Ok, yes let's do it this way for now.